src/App.js

```diff
@@ -58,7 +58,7 @@ function App() {
       <h3>{anime.length} episodes airing today</h3>
       <List anime={anime}/>
       <button onClick={() => setAnime([])}>
-        clear al
+        clear all
       </button>
     </section>
```

Comment:
make global variables

Reply:
@ShreyasSubhedar the said changes have been made.